packages/medusa/src/api/admin/orders/[id]/transfer/cancel/route.ts

```diff
@@ -0,0 +1,37 @@
+import { cancelOrderTransferRequestWorkflow } from "@medusajs/core-flows"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "@medusajs/framework/http"
+import { HttpTypes } from "@medusajs/framework/types"
+import { AdminCancelOrderTransferRequestType } from "../../../validators"
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/framework/utils"
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminCancelOrderTransferRequestType>,
+  res: MedusaResponse<HttpTypes.AdminOrderResponse>
+) => {
+  const remoteQuery = req.scope.resolve(ContainerRegistrationKeys.REMOTE_QUERY)
+
+  const orderId = req.params.id
+  const userId = req.auth_context.actor_id
+
+  await cancelOrderTransferRequestWorkflow(req.scope).run({
+    input: {
+      order_id: orderId,
+      logged_in_user_id: userId,
+    },
+  })
+
+  const queryObject = remoteQueryObjectFromString({
+    entryPoint: "order",
+    variables: { id: orderId },
+    fields: req.remoteQueryConfig.fields,
+  })
+
+  const [order] = await remoteQuery(queryObject)
```

Comment:
sugg: We have the new graph query that you can use. Eg: https://github.com/medusajs/medusa/blob/develop/packages/medusa/src/api/admin/orders/%5Bid%5D/line-items/route.ts#L19-L28